molecule/ci/tests/test_basic.py

```diff
@@ -1,4 +1,4 @@
-def test_mainpage(Command):
+def test_mainpage(host):
```

Comment:
Was not familiar with the `host` module—handy! :+1:

Reply:
You start to get deprecation warnings if you don't use `host` on recent `testinfra` versions